src/ClientHandle.h

```diff
@@ -453,7 +453,12 @@ class cClientHandle  // tolua_export
 		// TODO: Add Kicking here as well
 	} ;
 	
+#ifdef ANDROID
+#pragma message "Android <atomic> support is incomplete or broken - using plain variable (subject to data races!)"
+	eState m_State;
```

Comment:
Unless you want to review all the code that uses std::atomic, and come up with alternative implementations, I think a custom minimal atomic implementation is easier.